pages/c/_cluster/settings/brand.vue

```diff
@@ -21,14 +22,22 @@ const parse = require('url-parse');
 
 const DEFAULT_BANNER_SETTING = {
   bannerHeader: {
-    background: null,
-    color:      null,
-    text:       null
+    background:      null,
+    color:           null,
+    textAlignment:   null,
```

Comment:
Overall this looks fine and it works correctly. My only request is that you add a default value for `textAlignment` so that this radio group has something selected initially:
![Screen Shot 2021-08-20 at 1 23 50 PM](https://user-images.githubusercontent.com/42977925/130289788-09c2940d-4abb-4023-aaa9-b11c32994e88.png)

It looks like changing `textAlignment: null` to `textAlignment: 'center'` for header and footer defaults is sufficient.